pytests/src/pyclasses.rs

```diff
@@ -104,6 +106,34 @@ impl ClassWithDict {
     }
 }
 
+#[cfg(not(any(Py_LIMITED_API, PyPy, GraalPy)))] // Can't subclass native types on abi3 yet, PyPy/GraalPy don't support `py_super`
```

Comment:
We might want to make `py_super()` work by calling the python builtin directly on those interpreters? Maybe a follow up task depending what else we decide regarding that API.

Reply:
I've exposed it in 70981d7. If you agree I think this is ready to land then.